{"error": null, "issues": [{"address": 518, "debug": "block_number: 0xe58141feffeb2353fee8c10cb914dd7a8b0a83c244b22132ad451c07c217dbc5\ncallvalue: 0x20f459e0c00a4aa9810bf3eca71615b087b438f8fafb7e003155020c600713f8\nretval_357: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\nstorage_keccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\ncaller: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\ncalldata_EasyInvest6_0: 0xaac136200000000000000000000000000000000000000000000000000000000\ncalldatasize_EasyInvest6: 0x4\n", "description": "A possible integer overflow exists in the function `fallback`.\nThe addition or multiplication may result in a value higher than the maximum representable integer.", "filename": "/temp/Easy_Invest_6%-0.sol", "function": "fallback", "lineno": 45, "title": "Integer Overflow ", "type": "Warning"}], "success": true}
